packages/grid/x-grid-modules/src/models/gridOptions.tsx

```diff
@@ -98,17 +99,34 @@ export interface GridOptions {
    * Set the number of rows in one page.
    * @default 100
    */
-  paginationPageSize?: number;
+  pageSize?: number;
   /**
    * Auto-scale the pageSize with the container size to the max number of rows to avoid rendering a vertical scroll bar.
    * @default false
    */
-  paginationAutoPageSize?: boolean;
+  autoPageSize?: boolean;
   /**
-   * Select the paginationPageSize dynamically using the component UI.
+   * Select the pageSize dynamically using the component UI.
    * @default [25, 50, 100]
    */
-  paginationRowsPerPageOptions?: number[];
+  rowsPerPageOptions?: number[];
+
+  /**
+   * Pagination can be processed on the server or client side.
+   * Set it to FeatureMode.Client or `Client` if you would like to handle the pagination on the client side.
+   * Set it to FeatureMode.Server or `Server` if you would like to handle the pagination on the server side.
```

Comment:
```suggestion
   * Set it to FeatureMode.Client or `client` if you would like to handle the pagination on the client-side.
   * Set it to FeatureMode.Server or `server` if you would like to handle the pagination on the server-side.
```

Reply:
The casing is important in the string

Reply:
Sorry, I should have linked https://github.com/mui-org/material-ui-x/pull/100#discussion_r459576833